examples/vello_editor/src/text.rs

```diff
@@ -39,6 +41,27 @@ impl Editor {
         self.editor.text()
     }
 
+    pub fn init(&mut self) {
+        self.start_time = Some(Instant::now());
+        self.blink_period = Duration::from_millis(500);
```

Comment:
We should have a comment here that for simplicity we don't get the system blink period, but in real systems you *must* do so.

We can probably link to [#glazier > Cursor Blink Interval](https://xi.zulipchat.com/#narrow/channel/351333-glazier/topic/Cursor.20Blink.20interval)